packages/ts-morph/src/compiler/types/Type.ts

```diff
@@ -389,6 +389,13 @@ export class Type<TType extends ts.Type = ts.Type> {
     return symbol.getName() === "Array" && this.getTypeArguments().length === 1;
   }
 
+    /**
+     * Gets if this is a boolean type.
```

Comment:
template literal. I'll fix it after merge (going to do a release)

Reply:
Missed that 😅